content/docs/studio/user-guide/projects-and-experiments/live-metrics.md

```diff
@@ -0,0 +1,102 @@
+# Generate live (real-time) metrics for experiments
+
+In your model training CI action, you can use [DVCLive] to send live updates to
+metrics and plots back to Iterative Studio, without writing them to your Git
+repository. This requires a 3-step process:
+
+1. [Set up an access token](#set-up-an-access-token)
+2. [Configure your model training CI job](#configure-your-model-training-ci-job)
+3. [Send and view the updates](#send-the-updates)
+
+Live metrics must be sent to this ingestion end-point: TODO: Add the end-point
+here
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
+TODO: Replace this with a video tutorial for live metrics
+
+## Set up an access token
+
+Iterative Studio uses access tokens to authorize [DVCLive] to send live updates
+to the metrics and plots.
+
+### Generate access token
+
+To set up the access token, open your user profile page. In the
+`Studio access token` section, click on `Generate new token`.
+
+### Regenerate access token
+
+You can regenerate the access token at any point. The old access token will no
+longer be authorized to send live metrics and plots to Iterative Studio.
+
+### Reset (delete) access token
+
+You can delete the access token when you no longer need it.
+
+The option to delete the access token is also available when you try to change
+your password. That is, you can reset all your access credentials (your password
```

Comment:
Reset or delete?

Reply:
It's called reset in one place (change password modal) and delete in another (profile page) in Studio. That is why I used both the terms in the docs. But I guess it's better to fix this in Studio itself.
@mvshmakov can we change to one term everywhere - I think delete is clear, wdyt?

Reply:
Reset sounds like something is preserved but returned to its original condition. I think we mean delete 🙂

Reply:
I think so too. cc @mvshmakov

Reply:
Yes, within the flow of changing the password and flow of confirming the new password after resetting it, we have the option to delete an access token.

We have a correct label in a tooltip hint that states the token will be deleted. I've also renamed the "Reset access token" to "Delete access token" in both places in the app for clarity here: https://github.com/iterative/studio/pull/4800.

So, in the docs, we need to state "delete the access token".

Reply:
Now, we don't have a separate section for `Reset / delete access token`. And I've reviewed the text in this page for clarity.